homeassistant/components/soma/cover.py

```diff
@@ -0,0 +1,81 @@
+"""
+Support for Soma Covers.
+
+For more details about this platform, please refer to the documentation at
```

Comment:
Please remove this line and just keep the very first line.

Reply:
Fixed